src/containers/NFT/NFTHeader/Details.tsx

```diff
@@ -55,6 +55,10 @@ export const Details = ({ data }: Props) => {
           <td className="col1">{t('transfer_fee')}</td>
           <td className="col2">{formattedFee}</td>
         </tr>
+        <tr className="row">
```

Comment:
I think we should only show the row if burned.

Reply:
i'm ambivalent about this.

Reply:
hmm am I going to change it?